packages/build/tests/plugins/constants/snapshots/tests.js.md

```diff
@@ -161,7 +97,7 @@ Generated by [AVA](https://ava.li).
     │ 1. Running onInit command from netlify-plugin-test │␊
     └────────────────────────────────────────────────────┘␊
     ␊
-    .netlify/build␊
+    /file/path true␊
```

Comment:
The test snapshot shows that the default value changed from `.netlify/build` to the package root directory. This is normalized to `/file/path` in the snapshot since the file path is machine-specific which would make the snapshot not reproducible otherwise.